resources/istio/charts/gateways/values.yaml

```diff
@@ -109,14 +112,30 @@ istio-ingressgateway:
     name: tcp-dns-tls
   ####### end MESH EXPANSION PORTS ######
   ##############
-  secretVolumes: {}
+  secretVolumes:
```

Comment:
We are using SecretDiscoverySecrets (SDS), this volumes should not be added

Reply:
done